LayoutTests/fast/forms/select/select-show-picker.html

```diff
@@ -29,11 +29,28 @@
     testRunner.dumpAsText();
 }
 
+function getFormControlInteractionScript()
+{
+    return `
+    (function() {
+        uiController.didStartFormControlInteractionCallback(function () {
+            uiController.uiScriptComplete();
+        })
+    })();`
+}
+
 async function runTest() {
     await UIHelper.activateAt(0, 0);
-    select.showPicker();
+    select.showPicker?.();
 
-    before.textContent = internals.isSelectPopupVisible(select);
+    if (UIHelper.isIOSFamily()) {
+        await new Promise((resolve) => {
+            testRunner.runUIScript(getFormControlInteractionScript(), resolve);
```

Comment:
Not convinced this is actually testing anything meaningful when I disabled the feature flag locally for the test it still seemed to pass